.gitignore

```diff
@@ -13,4 +13,5 @@ gwt-unitCache
 .idea
 npm-debug.log
 .metadata/*
-*.log
\ No newline at end of file
+*.log
+manager/ui/war/apiman/translations.js
```

Comment:
It is always regenerated by the build, so there is no need to store it in git. Also, if it is in git, then it is possible for someone to think that they can update it instead of the properties file.